src/components/Agreement/mock-data.js

```diff
@@ -0,0 +1,20 @@
+// TODO: Remove and replace with connect data
+const connectedApp = {
+  appName: 'Voting',
+  collateralToken: {
+    address: '0x6b175474e89094c44da98b954eedeac495271d0f',
+    decimals: 18,
+    name: 'DAI Token',
+    symbol: 'DAI',
+  },
+  actionAmount: '100000000000000000000',
+  challengeAmount: '200000000000000000000',
+  challengeDuration: 172800000,
+}
```

Comment:
This is based on the outdated schema for the client, I wanted to port this over quickly and it'll be easier to update when we do the connect work.